semantic_conventions/trace/http.yaml

```
@@ -105,7 +105,7 @@ groups:
       - any_of:
           - [http.url]
           - [http.scheme, http.host, http.target]
-          - [http.scheme, net.peer.name, net.peer.port, http.target]
+          - [http.scheme, net.transport, net.peer.name, net.peer.port, http.target]
```

```suggestion
          - [http.scheme, net.peer.name, net.peer.port, http.target]
          - [http.scheme, net.peer.name, net.transport, http.target]
```

Otherwise we still require the port in any case.